web-client/public/js/grnstate.js

```diff
@@ -31,16 +32,20 @@ export const grnState = {
 
     newNetwork: false,
 
+    normalizationMax : null,
```

Comment:
The linter didn't catch this? Typically we don't have a space before colons in property-value lists.